docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+   #. ``.Values.server.config.query.maxMemory`` is set to ``4GB``.
+      This sets the :ref:`prop-resource-query-max-memory` property.
+   #. ``.Values.server.config.query.maxMemoryPerNode`` is set to ``1GB``.
+      This sets the :ref:`prop-resource-query-max-memory-per-node` property.
+   #. ``.Values.coordinator.jvm.maxHeapSize`` is set to ``8GB``.
+      This sets the ``Xmx`` property in the :ref:`jvm_config` on the
+      coordinator.
+   #. ``.Values.worker.jvm.maxHeapSize`` is set to ``8GB``.
+      This sets the ``Xmx`` property in the :ref:`jvm_config` on the
+      worker.
+
+      .. warning::
+         When updating memory configurations, you need to make sure that
+         :ref:`prop-resource-query-max-memory` is greater than
+         :ref:`prop-resource-query-max-memory-per-node`. Also make sure that the
+         sum of :ref:`prop-resource-query-max-memory-per-node` and
+         :ref:`prop-resource-memory-heap-headroom-per-node` is less than the
+         ``Xmx`` property in the :ref:`jvm_config`.
+
+
+#. Add the Trino Helm chart repository to Helm if you haven't done so already.
+   This tells Helm where to find the Trino charts. You can name the repo
+   whatever you want but we suggest using ``trino``.
+
+   .. code-block:: text
+
+       helm repo add trino https://trinodb.github.io/charts
+
+#. Install Trino on the Kubernetes cluster using the Helm chart. This generates
+   the Kubernetes configuration files by inserting properties into helm
+   templates. Helm requires default values but can be optionally overridden by a
+   ``values.yaml`` file to update default behavior. Run the following command to
+   use all default values and create a cluster called ``my-trino-cluster``.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   To override the default values, you can use the ``values.yaml`` created in
+   the previous step.
+
+   .. code-block:: text
+
+       helm install -f values.yaml my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output will differ based on your ``values.yaml`` configuration and
+   cluster name. For example, the port ``8080`` is set by the
+   ``.Values.service.port`` in the ``values.yaml``.
+
+#. Run the following command to check that all pods, deployments, and services are
+   running properly.
+
+   .. code-block:: text
+
+      kubectl get all
```

Comment:
So far in this doc most commands were using some optional namespaces and contexts. Maybe we could use something like this here, a namespace or a label selector to make sure we only list resources owned by the Trino chart?

Reply:
ah, should use a context yes thanks